.buildkite/pipeline.provision.yml

```diff
@@ -90,9 +120,6 @@ steps:
     build:
       commit: "${BULIDKITE_COMMIT}"
       branch: "${BUILDKITE_BRANCH}"
-
-  - wait
-
```

Comment:
This `wait` definitely needs to be present.

Reply:
Added back